ForemWebView/ForemWebView.swift

```diff
@@ -138,6 +142,14 @@ open class ForemWebView: WKWebView {
         }
     }
 
+    // Function that handles the swipe back from within the WebView for a more native fell navigation
+    @objc func swipeBack() {
+        guard let path = url?.path, foremInstance != nil else { return }
+        if path != "/" && canGoBack {
+            goBack()
+        }
+    }
+
```

Comment:
Is there any way that this can be written purely in swift?

Reply:
It is written purely in swift but in order for the method to be called by a registered observer (or available to other Objective C code) it is marked with the `@objc` attribute ([more on this here](https://www.hackingwithswift.com/example-code/language/what-is-the-objc-attribute#:~:text=That's%20where%20the%20%40objc%20attribute,as%20well%20as%20Swift%20code.))